frontend/src/lib/components/LemonCalendar/LemonCalendarRange.tsx

```diff
@@ -0,0 +1,146 @@
+import { LemonCalendar, LemonCalendarProps } from 'lib/components/LemonCalendar/LemonCalendar'
+import React, { useEffect, useState } from 'react'
+import { dayjs } from 'lib/dayjs'
+import { LemonButton } from 'lib/components/LemonButton'
+import { IconClose } from 'lib/components/icons'
+import { formatDateRange } from 'lib/utils'
+
+export interface LemonCalendarRangeProps {
+    value?: (string | null)[]
+    onChange: (date: string[]) => void
+    months?: number
+    getLemonButtonProps?: LemonCalendarProps['getLemonButtonProps']
```

Comment:
This seems very flexible, but I wonder what's the purpose? I can definitely imagine this being useful for range selection, but if it's just that, we shouldn't expose those nitty gritty details of individual button props for everyday usage.

Reply:
We weren't even using this :)